client/config/crd/populator.storage.k8s.io_volumepopulators.yaml

```diff
@@ -4,8 +4,8 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.2.4
-    api-approved.kubernetes.io: unapproved
+    controller-gen.kubebuilder.io/version: v0.5.0
+    api-approved.kubernetes.io: https://github.com/kubernetes-csi/volume-data-source-validator/pull/0
```

Comment:
pull 0 doesn't exist?  Probably better to leave it blank and leave a comment that it was committed without a pull number.  Maybe you can add the commit id in the comment for reference.

Reply:
It has to be the word "unapproved" or a URL pointing to a document (enforced by API server). I suppose I can point to the KEP document.

Reply:
Sure.